docs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "docs",
+  "version": "0.0.1",
+  "description": "Docs page for React-Chatbot",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "docs": "node docsServer.js",
```

Comment:
Can we centralize all scripts in the root package.json? For example we have `docs` and `buildDocs` in the root package.json for react-search: https://github.com/vectara/react-search/blob/main/package.json#L11